### PR TITLE
Add Index and Create book routes

### PR DESCRIPTION
**Pull request by juanisierra**
# Summary

* Added BooksController
* Added index and Create routes

# Trello Cards

https://trello.com/c/RjU7mmze/24-crear-rutas-index-y-show-para-book

### CODE REVIEW

File: app/serializers/book_serializer.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class BookSerializer < ActiveModel::Serializer
+  attributes :id, :author, :title, :publisher, :year, :genre
```
Comment (hdf1986) [inline app/serializers/book_serializer.rb]: The response isn't the same than the one in apiary, check: http://docs.wbooksapi.apiary.io/#reference/books/collection/create and try until you get the same response (check the image key)

File: app/controllers/api/v1/books_controller.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        render json: @books, status: :ok
+      end
+
+      def create
+        @book = Book.new(book_params)
+
+        if @book.save
+          render json: @book
+        else
+          render json: @book.errors
```
Comment (hdf1986) [inline app/controllers/api/v1/books_controller.rb]: Render this under the "errors" key

Comment (hdf1986) [inline app/controllers/api/v1/books_controller.rb]: And add the unprocessable entity status

File: app/controllers/api/v1/books_controller.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      def index
+        @books = Book.all
+        render json: @books, status: :ok
+      end
+
+      def create
+        @book = Book.new(book_params)
+
+        if @book.save
+          render json: @book
```
Comment (hdf1986) [inline app/controllers/api/v1/books_controller.rb]: Remember to render this under the "book" key

**Comment by juanisierra** [comment]
Fixed!

**Pull request merged**